packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -46,16 +46,30 @@ describe('Steps', () => {
     cy.get(setupComponentStep).should('not.exist')
   })
 
-  it('shows all e2e steps for an e2e project with custom testFiles', () => {
-    cy.scaffoldProject('migration-e2e-custom-test-files')
-    cy.openProject('migration-e2e-custom-test-files')
+  // TODO: the current logic is wrong and does not consider
+  // the case of a custom integration folder!
```

Comment:
I see this test and I see it using a custom `integrationFolder` (`src`). 
I see that files are renamed but left in the same folder.
This feels right.

Can you be more explicit? What logic is wrong? What do we need to consider?

Reply:
Oh, the comment was from before (when it was wrong). This PR fixes the wrong. I will remove the comment.